webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-grading.js

```diff
@@ -391,6 +400,40 @@ export class SakaiRubricGrading extends RubricsElement {
       console.log(textStatus);console.log(errorThrown);
     });
   }
+
+  pdfRubric(e) {
+    const reportRequestUrl = new URL('http://' + window.location.host + '/rubrics-service/rest/rubrics/generatePdf');
```

Comment:
You're harcoding the protocol here, it could be https://

Reply:
You don't need the protocol or the host. You should just be able to hit '/rubrics-service/rest/rubrics/generatepdf'